aten/src/ATen/native/SegmentReduce.cpp

```diff
@@ -60,6 +61,54 @@ Tensor _segment_reduce_cpu_kernel(
   return output;
 }
 
+Tensor _segment_reduce_cpu_backward_kernel(
+    const Tensor& grad,
+    const Tensor& output,
+    const Tensor& data,
+    const Tensor& lengths) {
+  auto grad_output = at::zeros({data.sizes()}, grad.options());
```

Comment:
This variable should be called `grad_input` (it is gradient with respect to operation input), `grad_output` or `grad` is the gradient you receive

Reply:
Sounds good, updated.